### ISSUE TITLE
Updated scripts have problems on NX-OS devices

### ISSUE THREAD

**Issue by jamiecaesar** [jamiecaesar/SecureCRT#8]
Particular 7K running 6.0(4) over console.  Need to verify with Telnet, but problem seems to be additional whitespace characters.

**Comment by jamiecaesar** [comment]
There were extra "\r"s following the echo of the command output, so waiting on the prompt + "\r\n" wasn't working.  Now just wait for just the prompt, and don't save any lines that only contain "\r" (which are the remnants after the prompt).  The rest of the logic works fine.

**Comment by garibaldi0** [comment]
Ah...was just about to try a test.  Glad you got it.

**Comment by jamiecaesar** [comment]
I also made some changes to the Document_Device script.   I added a folder
(since it creates so many files) and I changed the file format so that you
see the command name first.  I left the hostname in the filename, just in
case people wanted to copy files into a different directory.   I decided to
make the change today after noticing in OSX Finder that all the files
looked about the same (because it put ... in the middle).   Now I can see
the command name and the hostname is hidden under ... -- which is fine
because I'm in the folder named after the host.

Would be interested to hear if you disagree  (also DN0000, although I don't
know if he's getting notified on these, since he wrote the original script).
